tutorials/tour/basics.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: tutorial
+title: Basics
+
+disqus: true
+
+tutorial: scala-tour
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
+---
+
+In this page, we will cover basics of Scala.
+
+## Running Scala on Browser
+
+You can run Scala on browser by using [ScalaFiddle](https://scalafiddle.io).
+
+1. Go to https://scalafiddle.io.
+2. Copy and paste `println("Hello, world!")` to the left pane.
+3. Hit "Run" button and see it prints "Hello, world!" on the right pane.
+
+It is a perfect way for anybody to experiment a piece of Scala code anywhere, anytime!
+
+## Variables
+
+You can define variables with the `val` keyword.
+
+```
+val x = 1 + 1
+```
+
+Variables defined by `val` cannot be re-assigned and are immutable in that sense.
+
+```
+val x = 1 + 1
+x += 2 // This does not compile because it is re-assigning the variable.
+```
+
+If you need to mutate variables, you can use the `var` keyword instead.
+
+```
+var y = 1 + 1
+y += 2 // This compiles because "y" is declared with "var" keyword.
```

Comment:
It might be better here and in the case above to use simple assignment like `y = 42` rather than the compound operation `+=`. Or at least desugar it to `y = y + 2`.